### PR TITLE
Remove DOWNLEVEL_PRIOR_TO_WIN8 flag

### PR DESCRIPTION
**Pull request by huichen123**
- remove DOWNLEVEL_PRIOR_TO_WIN8/WIL_SUPPRESS_PRIVATE_API_USE/STANDaLONEDLL_EXPORTS flags
- hook up telemetry for mrmmin/mrmex code
- some improvements to telemetry.

### CODE REVIEW

**Comment by rohanp-msft** [comment]
You're removing code that's included if DOWNLEVEL_PRIOR_TO_WIN8 is NOT defined. Shouldn't we keep that code? And instead remove code that's included if DOWNLEVEL_PRIOR_TO_WIN8 IS defined?

**Comment by huichen123** [comment]
That's the correct behavior. The OSS code uses DOWNLEVEL_PRIOR_TO_WIN8 defined. 

---
In reply to: [910843591](https://github.com/microsoft/WindowsAppSDK/pull/1357#issuecomment-910843591) [](http://example.com/codeflow?ancestors=910843591,910843591)

**Review by rohanp-msft — APPROVED** [review]
:shipit:

File: dev/MRTCore/mrt/mrm/include/mrm/common/PlatformWin32.h [hunk 1]
```diff
@@ -95,72 +90,45 @@ extern "C"
...
-        LOG_ERROR_IN_MEMORY(hr, 2, nullptr, msg2); \
-        MrtRuntimeTraceLoggingProvider::MeasureGenericEventParam2(TOWIDE(__FUNCTION__), msg, RemoveUsernameFromPath(msg2), hr); \
+        LOG_ERROR_IN_MEMORY(hr, __LINE__, __FILE__, msg2); \
+        MrtRuntimeTraceLoggingProvider::MeasureGenericEvent(TOWIDE(__FUNCTION__), msg, RemoveUsernameFromPath(msg2), hr); \
     }
 
-#define WRITE_ETW(etw) etw
-
-#endif
+#define WRITE_ETW(etw) __noop
```
[the remark this answers is not in the document]
Comment (huichen123) [reply dev/MRTCore/mrt/mrm/include/mrm/common/PlatformWin32.h:134]: actually there is no change for this. it was "__noop". The code used before was the code being remove above. The telemetry code you added was not taking effect.

**Pull request merged**